tests/my_house_tests.py

```diff
@@ -148,3 +166,36 @@ def test_delete_expenses_of_my_house(self, client, house_factory, expense_factor
         assert len(Expenses.objects.all()) == 0
         assert response.status_code == 302
         assert response.url == '/../house'
+
+    def test_edit_existing_house(self, client, house_factory, new_user):
+        client.force_login(new_user)
+        country = Country.create_country(COUNTRY)
+        house = House.create_house(
+            user=new_user,
+            name=HOUSE_NAME,
+            public=HOUSE_PUBLIC,
+            country=country,
+            city=City.create_city(CITY, country),
+            parent_profession_1=HOUSE_PARENT_PROFESSION_1,
+            parent_profession_2=HOUSE_PARENT_PROFESSION_2,
+            income=HOUSE_INCOME,
+            children=HOUSE_CHILDREN_BEFORE,
+            description=HOUSE_DESCRIPTION,
+        )
+        house.save()
+        house_form = {
+            'public': HOUSE_PUBLIC,
+            'country': COUNTY_OR_CITY,
+            'city': COUNTY_OR_CITY,
+            'parent_profession_1': HOUSE_PARENT_PROFESSION_1,
+            'parent_profession_2': HOUSE_PARENT_PROFESSION_2,
+            'children': HOUSE_CHILDREN_AFTER,
+            'income': HOUSE_INCOME,
+            'name': 'house name',
+        }
+        response = client.post('/house_create/', data=house_form)
+        edited_house: House = House.objects.all()[0]
+        assert edited_house.children == HOUSE_CHILDREN_AFTER
+        assert len(House.objects.all()) == 1
+        assert response.status_code == 302
+        assert response.url == '/../house'
```

Comment:
Generic suggestion comment for all 302 use cases,
You can create new request based on 302 response url

Reply:
Is that your intention? I could not find another solution other than this
![image](https://user-images.githubusercontent.com/93659937/170499328-1652b465-1e74-4c82-89dc-4f4f321b3fd5.png)
@Yarboa

Reply:
You should take the content from 302 and run a get(response.uri)